Update installation.md (#49)
LycheeOrg/LycheeOrg.github.io: docs/installation.md

   		- Tokenizer
   		- XML
   		- ZIP

>    	- These PHP extensions are necessary if you are running a FreeBSD system:

Reviewer: I'd drop this line and merge the lists. Other extensions (e.g. JSON) are included by default, so we should probably list all of them.

   	 	- Simplexml
		- Dom
   	 	- Session
   	 	- Zlib
   	- For database, you will need these PHP extensions:
   	 	- SQLite3 (For Lychee's inbuilt SQLite3)

>    	 	- MySQLi (For MySQL or MariaDB)
>    	 	- PDO_MySQL (For MySQL or MariaDB)

Reviewer 2: ```suggestion
   	 	- MySQLi (For MySQL or MariaDB, recommended modern extension)
   	 	- PDO_MySQL (For MySQL or MariaDB, legacy fall-back extension in case MySQLi is not available)
```

The list sounds as if both extensions were required for MySQL simultaneously, but it is an either-or with bias towards MySQLi. I woner if there is any PHP version and or distribution which does not provide MySQLi, especially since our minimal supported version is PHP 8.

But I don't use FreeBSD myself, so I have no idea.

Author: During my installation, I install MySQLi only, and the installation check webpage is showing PDO_MySQL aslo required.

>    	- For database, you will need these PHP extensions:
>    	 	- SQLite3 (For Lychee's inbuilt SQLite3)
>    	 	- MySQLi (For MySQL or MariaDB)
>    	 	- PDO_MySQL (For MySQL or MariaDB)

Reviewer: Wording, and added pgsql as it wasn't mentioned. @nagmat84 Does this wording satisfy your preferred vs legacy concern, or would you prefer it to be more explicit?

```suggestion
   	- You will also need one of these PHP extensions:
   	 	- SQLite3 for SQLite3 databases
   	 	- MySQLi (or PDO_MySQL) for MySQL or MariaDB databases
   	 	- PgSql (or PDO_PGSQL) for PostgreSQL databases
```

> - You also need install composer to complete installation, visit https://getcomposer.org/download/ for help.

Reviewer: I'm not sure whether this is needed in this section, but if it's to be included here it should be more specific - installing without composer is possible (from a release).

```suggestion
- To install from git you will also need composer. See [below](#from-the-master-branch) for details.
```

Author: Yeah, I install from git because it failed from release at first.

##### Path to the Lychee public/ directory.
    root /var/www/Lychee/public/;

>     index index.php;

Reviewer: :+1:

        fastcgi_pass unix:/run/php/php8.1-fpm.sock;
        fastcgi_index index.php;

>         fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
> ######## You may need change #document_root to your public folder if you are running a FreeBSD system.
> 	# fastcgi_param SCRIPT_FILENAME /var/www/Lychee/public$fastcgi_script_name;

Reviewer: What we set here appears to be the default, so I'm not sure why we're setting it at all. Your comment feels confusing in its current form - will it not accept `$document_root`?

```suggestion
######## You may need to replace $document_root with the absolute path to your public folder.
        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
```

Author: Acutally I don't know if it's only applying for FreeBSD. But if I don't change $document_root to absolute path, Nginx will return "File not found".
I think maybe relative path has some bugs on FreeBSD. Because I also met a ffmpeg probelm, video donot have thumbs and has_ffmpeg, has_exiftools are both set to 0. I fix it by add absolute path of FFprobe in ../vendor/lychee-org/php-exif/lib/PHPExif/Adapter/FFprobe.php according to https://lycheeorg.github.io/docs/faq.html#why-dont-my-videos-have-thumbnails.
> protected $toolPath = '/usr/local/bin/ffprobe';

In addition, I don't have Lychee/app/ModelFunctions/PhotoFunctions.php (prior to 4.2.0) or Lychee/app/Actions/Photo/Extensions/VideoEditing.php (starting with 4.2.0), but ffmpeg works well after I fixed FFprobe path issue. Maybe from which new version, Lychee removed or changed it?
